### PR TITLE
The Value of Unguided Work - Sarah's blog post

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Add post
- Add og-image
- Add header visual

### CODE REVIEW

File: src/posts/2023-09-11-value-of-unguided-work.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: "The Value of Unguided Work"
+authorHandle: sarahlorenzu
+tags: []
```
Comment (marcoow) [inline src/posts/2023-09-11-value-of-unguided-work.md]: This needs a tag – maybe `culture` or sth.

File: src/posts/2023-09-11-value-of-unguided-work.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+Shopify introduced its Meeting Cost Calculator in July 2023, helping estimate
+the cost of meetings. By getting rid of three meetings a week per person, the
+company expects a 15% cost reduction. (Read
+[more](https://edition.cnn.com/2023/07/12/tech/shopify-meeting-cost-calculator/index.html))
+
+![Shopify's meeting cost calculator](/assets/images/posts/2023-09-11-value-of-unguided-work/shopify-meeting-cost-calculator.jpg)
+
+### Time for Experimentation and Learning
+
+Following the concept promoted by Google, we encourage our team members to spend
```
Comment (marcoow) [inline src/posts/2023-09-11-value-of-unguided-work.md]:
```suggestion
Following the concept promoted by Google, our team members spend
```

File: src/posts/2023-09-11-value-of-unguided-work.md (hunk 1 above)
Comment (emmasofiah2o) [reply src/posts/2023-09-11-value-of-unguided-work.md]: culture sounds good, I'll add that :)

File: static/assets/images/posts/2023-09-11-value-of-unguided-work/shopify-meeting-cost-calculator.jpg
Comment (SaeLLo) [inline static/assets/images/posts/2023-09-11-value-of-unguided-work/shopify-meeting-cost-calculator.jpg:1]: Can we put "Source: Spotify" underneath the picture?

File: src/posts/2023-09-11-value-of-unguided-work.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+constantly, with project management’s main role being to make sure things are on
+track in the right general direction.
+
+Solid developer tooling, infrastructure, and practices enable teams to
+experiment and fail without causing havoc. We’ve worked with many international
+teams over the years and helped them set up and improve automatic testing and
+deployment, incremental rollout strategies, quality assurance, and other
+mechanisms that can raise a team’s ability to experiment.
+
+Reach out to talk about how we can help your team:
```
Comment (SaeLLo) [inline src/posts/2023-09-11-value-of-unguided-work.md]: Maybe connecting the sentence with the contact link would look nicer?

**Comment by SaeLLo** [comment]
image

The formatting looks a bit off.

**Comment by SaeLLo** [comment]
Shall we use my newest picture? The one we are using here is quite old.

**Comment by emmasofiah2o** [comment]
> image
> The formatting looks a bit off.

I have no influence on the formatting, this is all pre-defined. The only thing we can do is shorten the text.

**Comment by emmasofiah2o** [comment]
> Shall we use my newest picture? The one we are using here is quite old.

Could you please send it to me? I took this from slack.

**Review by SaeLLo — APPROVED** [review]

**Review by SaeLLo — APPROVED** [review]

**Review by SaeLLo — APPROVED** [review]

**Comment by emmasofiah2o** [comment]
@marcoow Sarah's old picture is still showing. Bart suggested you might need to clear the cache on the Main branch now that it's merged. Could you please do that when you have a minute? :)

**Comment by marcoow** [comment]
rerunning deployment – should be fixed in 2min

**Pull request merged**